applications/ParticleMechanicsApplication/python_scripts/mpm_quasi_static_solver.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications and dependencies
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+
+# Importing the base class
+from KratosMultiphysics.ParticleMechanicsApplication.mpm_implicit_dynamic_solver import MPMImplicitDynamicSolver
+
+def CreateSolver(model, custom_settings):
+    return MPMQuasiStaticSolver(model, custom_settings)
+
+class MPMQuasiStaticSolver(MPMImplicitDynamicSolver):
```

Comment:
hmmm this seems strange
What is the difference to the static one?

Reply:
So the quasi-static is actually a dynamic simulation without damping and mass term in the system matrix. The flag _is_dynamic tells whether these two terms being computed or not computed.

Reply:
But that sounds like static to me just with some time steps
Check what is done in Structural

Reply:
I also think that we can do stepping within the static solver - this would simplify the application a bit.

Reply:
The MPM static solver uses the default strategy in the core, which is work actually the same with FEM. While in MPM we need to do extrapolation and interpolation after one timestep is over. Therefore, I am using the dynamic without the damping and mass matrix.